ibm/resource_ibm_atracker_route.go

```diff
@@ -0,0 +1,211 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/platform-services-go-sdk/atrackerv1"
+)
+
+func resourceIBMAtrackerRoute() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMAtrackerRouteCreate,
+		ReadContext:   resourceIBMAtrackerRouteRead,
+		UpdateContext: resourceIBMAtrackerRouteUpdate,
+		DeleteContext: resourceIBMAtrackerRouteDelete,
+		Importer:      &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(20 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
Can we have a validator added with some regex

Reply:
The API doc currently specifies no `pattern`, `minLength`, or `maxLength`.  If these are added, an appropriate validator will be generated.